src/pages/AllFestivals/AllFestivals.tsx

```diff
@@ -22,28 +27,71 @@ function AllFestivals() {
     getFestivals();
   }, []);
 
+  useEffect(() => {
+    async function clickedFestival() {
+      if (query != "") {
+        const response = await fetch(`/api/festivals/name/${query}`);
+        const body = await response.json();
+        setFestival(body);
+      }
+    }
+    clickedFestival();
+  }, [query]);
+
+  function close() {
+    setFestival("");
+  }
+
```

Comment:
Isn't the data for the festival also stored in the festivals array?

Reply:
Oh yes, you’re absolutely right! I’ll check this out, thanks!